File: src/Zone.java
//import org.json.*;

public class Zone {
    private String name;
    private Triangle trianglePosition;

    public Zone(String name, Triangle trianglePosition) {
            this.name = name;
            this.trianglePosition = trianglePosition;
        }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Triangle getTrianglePosition() {
        return this.trianglePosition;
    }

    public void setTrianglePosition(Triangle trianglePosition) {
        this.trianglePosition = trianglePosition;
    }

}

File: Python/Zone.py
class Zone:
    def __init__(self, name, triangle_position):
        self.name = name
        self.triangle_position = triangle_position

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_triangle_position(self):
        return self.triangle_position

    def set_triangle_position(self, triangle_position):
        self.triangle_position = triangle_position
